velox/exec/SpillFile.cpp

```diff
@@ -63,7 +63,7 @@ SpillWriteFile::SpillWriteFile(
     const std::string& pathPrefix,
     const std::string& fileCreateConfig)
     : id_(id), path_(fmt::format("{}-{}", pathPrefix, ordinalCounter_++)) {
-  auto fs = filesystems::getFileSystem(path_, nullptr);
+  auto fs = filesystems::getLocalFileSystem();
```

Comment:
We can't change like this. In Meta, we spill to remote storage.

Reply:
What scheme does your remote storage use? Does your remote storage not need any properties?